client/src/components/ProjectWizard/WizardPages/Level0Page.jsx

```diff
@@ -15,17 +15,17 @@ const useStyles = createUseStyles({
     textAlign: "center",
 
     "& h1": {
```

Comment:
Instead of hard-coding the font-related styles, this should reference the theme's typography.heading1 style